scalingo/data_scalingo_database_firewall_managed_range.go

```diff
@@ -0,0 +1,77 @@
+package scalingo
+
+import (
+	"context"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/Scalingo/go-scalingo/v8"
+)
+
+func dataSourceScDatabaseFirewallManagedRange() *schema.Resource {
+	return &schema.Resource{
+		ReadContext: dataSourceScDatabaseFirewallManagedRangeRead,
+		Description: "Database firewall managed range retrieved from the Database API",
+
+		Schema: map[string]*schema.Schema{
+			"database_id": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "ID of the Database NG",
+			},
+			"name": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "Name of the managed range to look up",
```

Comment:
question: what is a "managed range"? Is it a typo?
```suggestion
				Description: "Name of the managed database to look up",
```